documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -70,7 +77,6 @@ Release Notes
 7.1.26
 ======
 * Released with AVX disabled.
-* Fixed status json to not report fdbserver processes as clients. `(PR #9252) <https://github.com/apple/foundationdb/pull/9252>`_
```

Comment:
Why did we remove this one? Just curious

Reply:
maybe not intended to remove?